### PR TITLE
add style-component in Footer

### PR DESCRIPTION
**Pull request by Momcil**
add style-component in Footer

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 function Footer() {
   return (
     <div className='Navbar'>
-    <ul>
+    <NavContainer>
       <li className='Home/Dashboard'><a href='#'>Dashboard</a></li>
       <li className='Ranrom Characters'><a href='#'>Ranrom</a></li>
       <li className='Favorites'><a href='#'>Favorites</a></li>
-    </ul>
+    </NavContainer>
```
Comment (AlexF090) [inline src/components/Footer.jsx:11]: I linke :D

File: src/App.css [hunk 2]
```diff
@@ -33,14 +33,7 @@
...
   color: white;
 }
-ul{
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-  list-style-type: none;
-  background-color: #282c34;
-  width: auto;
-}
```
Comment (AlexF090) [inline src/App.css:43]: Perfect

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
 }
 
+const NavContainer = styled.ul`
+display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  list-style-type: none;
+  background-color: #282c34;
+  width: auto;
+`
```
Comment (AlexF090) [inline src/components/Footer.jsx:23]: Top

**Review by AlexF090 — APPROVED** [review]

File: src/components/Footer.jsx (hunk 3 above)
Comment (Momcil) [reply src/components/Footer.jsx:23]: tanks Alex !

**Review by ThomasF85 — COMMENTED** [review]
Good job! But the Issue is not closed yet, since you only have used styled components for the footer. Please only start a PR once an Issue is done.

**Pull request merged**